### PR TITLE
add FAQ to Publish Form

### PR DESCRIPTION
**Pull request by ykris45**
## Preview
image

### CODE REVIEW

**Comment by tzarebczan** [comment]
`Our` should be lowercase. @seanyesmunt / @skhameneh what do you think about the wording/placement?

File: src/renderer/component/publishForm/view.jsx [hunk 1]
```diff
@@ -402,12 +409,12 @@ class PublishForm extends React.PureComponent<Props> {
               {uploadThumbnailStatus === THUMBNAIL_STATUSES.API_DOWN ? (
                 __('Enter a URL for your thumbnail.')
               ) : (
-                <React.Fragment>
-                  {__('Upload your thumbnail (.png/.jpg/.jpeg/.gif) to')}{' '}
-                  <Button button="link" label={__('spee.ch')} href="https://spee.ch/about" />.{' '}
-                  {__('Recommended size: 800x450 (16:9)')}
-                </React.Fragment>
-              )}
+                  <React.Fragment>
```
Comment (skhameneh) [inline src/renderer/component/publishForm/view.jsx]: Please fix the excess indentations (is your IDE set correctly?)

File: src/renderer/component/publishForm/view.jsx [hunk 2]
```diff
@@ -482,7 +489,7 @@ class PublishForm extends React.PureComponent<Props> {
                     !channel || channel === CHANNEL_ANONYMOUS || channel === CHANNEL_NEW
                       ? ''
                       : `${channel}/`
-                  }`}
+                    }`}
```
Comment (skhameneh) [inline src/renderer/component/publishForm/view.jsx]: Remove excess indentation

Comment (ykris45) [reply src/renderer/component/publishForm/view.jsx]: on line 492 sir

**Comment by ykris45** [comment]
@tzarebczan bingo

**Pull request merged**